packages/blade/src/components/Typography/index.ts

```diff
@@ -1,6 +1,6 @@
-export { default as Title } from './Title';
+export { Title } from './Title';
```

Comment:
Why do we need 
```jsx
export { Title } from './Title';
```
if we're also doing
export * from './Title';

Same for Heading & Text

Reply:
wait, i fixed this already i think on my local, could be diff branch. let me fix this